tools/commands/build.py

```diff
@@ -127,6 +132,33 @@ def make(self):
         logger.info("building & installing {}: {}".format(self.name, " ".join(cmd)))
         subprocess.check_call(cmd, env=self.env)
 
+# hostap builder class
+class hostapbuilder(builder):
+    def __init__(self, name, modules_dir, install_dir, make_verbose=False):
+        self.make_verbose = make_verbose
+        super(hostapbuilder, self).__init__(name, modules_dir, modules_dir, install_dir)
+        self.config_src_fpath = os.getcwd() + "/config/" + name + ".config"
+        self.config_dst_fpath = self.build_path + "/.config"
+
+    def clean(self):
+        os.system("make -C {} clean".format(self.build_path))
+        if os.path.exists(self.config_dst_fpath):
+            os.remove(self.config_dst_fpath)
+
+    def prepare(self):
+        logger.info("preparing {}".format(self.name))
+        shutil.copy2(self.config_src_fpath, self.config_dst_fpath)
+
+    def make(self):
+        bin_dir = "BINDIR=" + self.install_path + "/bin"
+        cmd = ["make", bin_dir, '-C', self.src_path, "install"]
```

Comment:
I know they're the same, but use build_path here:
```suggestion
        cmd = ["make", bin_dir, '-C', self.build_path, "install"]
```